commands/cluster-slot-stats.md

```diff
@@ -0,0 +1,99 @@
+`CLUSTER SLOT-STATS` returns an array of slot usage statistics for slots assigned to the current shard.
```

Comment:
I just realize we kind of use metrics and statistics sort of interchangeably which now bothers me, but maybe it's fine.

Reply:
The command name is the title of the page, so it doesn't need to here.

```suggestion
Returns an array of slot usage statistics for slots assigned to the current shard.
```